slo_generator/backends/dynatrace.py

```diff
@@ -201,7 +246,10 @@ def retry_http(response):
         bool: True to retry, False otherwise.
     """
     retry_codes = [429]
-    code = int(response.get('error', {}).get('code', 200))
+    if isinstance(response.get('error', {}), str):
+        code = 200
+    else:
+        code = int(response.get('error', {}).get('code', 200))
```

Comment:
can we avoid repeating `response.get('error', {})` by setting it as a variable before the `if` / `else` ?